cloudinit/net/network_state.py

```diff
@@ -58,38 +58,6 @@
                'bridge_waitport': None}}
 
 
-def parse_net_config_data(net_config, skip_broken=True):
```

Comment:
I moved `parse_net_config_data` to the bottom of the file so I could return some type information (`NetworkConfig` isn't defined here yet).

I removed `parse_net_config` as it is dead code.